content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog
 title: "k8s.gcr.io image registry will be frozen from the 3rd of April 2023"
-date: 2023-02-06
+date: 2023-02-06T17:00:00
 slug: k8s-gcr-io-freeze-announcement
```

Comment:
nit: Can you make the title sentence case?
title: "k8s.gcr.io Image Registry Will Be Frozen From the 3rd of April 2023"